tests/README.md

```diff
@@ -1,24 +1,115 @@
 # Integration tests
 
-Integration tests require a running Kubevirt cluster.  Once you have a running
-Kubevirt cluster, you can use the `-master` and the `-kubeconfig` flags to
-point the tests to the cluster.
+Kubevirt integration tests are a suite of end-to-end tests which run
+against an active Kubevirt cluster.
 
-## Run them on an arbitrary KubeVirt installation
+The tests may execute against various clusters, from integrated kubevirtci
+deployments and up to external kubernetes or OpenShift ones.
+
+## Kubevirtci
+
+[Kubevirtci](https://github.com/kubevirt/kubevirtci/)
+provides containerized Kubernetes clusters which can be used
+to test KubeVirt and similar projects.
+
+KubeVirt integration tests are being run against kubevirtci clusters on CI
+and is the simplest way to run the tests locally in a development environment.
 
+To run the test locally:
+- Configure the desired cluster setup (provider, number of nodes and nics).
+
+  Note: Some tests may run only on specific configuration and hardware.
+
+  Note: For detailed instructions on how to work with a specific provider,
```

Comment:
There is no one way, but at least in [kubevirt-user-guide](https://github.com/kubevirt/user-guide/blob/master/creation/interfaces-and-networks.md#ports), many notes are in a like quotation style (with >) , ,with the word "Note" in **bold**. 
So maybe consider something like : 
```
 > **Note:** For detailed instructions on how to work with a specific provider, please explore kubevirtci providers, e.g.
  [k8s-1.18](https://github.com/kubevirt/kubevirtci/tree/master/cluster-up/cluster/k8s-1.18)
```

Reply:
Done